src/App.css

```diff
@@ -29,8 +29,26 @@ ul{
   margin: 10px
 }
 
+.single_review_image{
+  width: 80%;
+  height: 350px;
+}
+
 @media only screen and (min-width: 768px) {
 
+  .App {
+    text-align: center;
+    background-color: rgb(61, 136, 71);
+    font-variant: small-caps;
+    font-family: monospace;
+    font-size:xx-large;
+  }
+
+  .single_review_image{
```

Comment:
This is causing the image to stretch on certain screen sizes. Have a look at the CSS property object-fit, as it should help 👍